packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -857,7 +857,7 @@ class Component
       if (!lazyEventsProcessedForComponentClass.has(proto)) {
         let lazyEvents = eventDispatcher.lazyEvents;
 
-        lazyEvents.forEach((mappedEventName: string, event: string) => {
+        lazyEvents.forEach((mappedEventName: string | null, event: string) => {
```

Comment:
Given the check on the next line we definitely expect a possible null here.

Reply:
Out of curiosity, does `lazyEvents` itself not have the right type? Seems like we want it to be there instead?

Reply:
Types are correct. Looks like this ends up being redundant.